### PR TITLE
use rustfmt 0.6.0

### PR DESCRIPTION
**Pull request by siddontang**
PTAL @BusyJay @zhangjinpeng1987

### CODE REVIEW

File: travis-build/Makefile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+prepare-rustfmt:
+	curl -L https://github.com/tennix/rustfmt/releases/download/v0.6/rustfmt-${RUSTFMT_VERSION}-linux-amd64.tar.gz -o rustfmt-${RUSTFMT_VERSION}-linux-amd64.tar.gz && \
+	mkdir -p ${HOME}/.cargo/bin && tar xzf rustfmt-${RUSTFMT_VERSION}-linux-amd64.tar.gz -C ${HOME}/.cargo/bin --strip-components=1
```
Comment (andelf) [inline travis-build/Makefile]: EOF

**Comment by siddontang** [comment]
PTAL

**Comment by zhangjinpeng1987** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**